packages/visx-shape/src/shapes/Pie.tsx

```diff
@@ -7,6 +7,8 @@ import { arc as arcPath, pie as piePath } from '../util/D3ShapeFactories';
 
 export type PieArcDatum<Datum> = PieArcDatumType<Datum>;
 
+type StringAccessor<Datum> = (arc: PieArcDatum<Datum>) => string;
```

Comment:
**Question**: I chose to provide `PieArcDatum` since it also includes the `index`, but it could also be `(datum: Datum, index: number)`. I'd like to get your thoughts on this - `PieArcDatum` may be unnecessary, especially if the pattern in other components is to merely provide the `Datum` type.

If not, I think the name shouldn't be `arc` but instead something like `arcDatum` or just `pieArcDatum` - I can change

Reply:
Good question, I think that `PieArcDatum` makes sense. Since it's strictly more info, including the computed arc size, it seems like a win to me.